apps/fabric-website/src/pages/Styles/OfficeBrandIconsPage/docs/web/OfficeBrandIconsSingleColor.md

```diff
@@ -1,4 +1,4 @@
-Fabric’s icon font set also contains single-color product and file type icons. You use these single-color icons like all other font-based icons in Fabric. For more information, see the [Fabric icons](#/styles/web/icons) page or refer to the following example:
+Fabric’s icon font set also contains single-color product and file type icons. You use these single-color icons like all other font-based icons in Fabric. For more information, see the [Fluent UI icons](#/styles/web/icons) page or refer to the following example:
```

Comment:
Are you going to get Fabric followed by a single quote, curly single quote, or period later?

Reply:
Yup...this one is specifically targeting Fabric + space to capture some of the compound Fabric words. next sweep is any fabric string